backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -297,11 +297,21 @@ public static Optional<Extension> convertToTribalAffiliationExtension(String tri
   }
 
   public static Practitioner convertToPractitioner(Provider provider) {
+    return convertToPractitioner(
+        provider.getInternalId().toString(),
+        provider.getNameInfo(),
+        provider.getTelephone(),
+        provider.getAddress(),
+        DEFAULT_COUNTRY);
+  }
+
+  public static Practitioner convertToPractitioner(
+      String id, PersonName name, String telephone, StreetAddress addr, String country) {
     var practitioner = new Practitioner();
-    practitioner.setId(provider.getInternalId().toString());
-    practitioner.addName(convertToHumanName(provider.getNameInfo()));
-    practitioner.addAddress(convertToAddress(provider.getAddress(), DEFAULT_COUNTRY));
-    practitioner.addTelecom(convertToContactPoint(ContactPointUse.WORK, provider.getTelephone()));
+    practitioner.setId(id);
+    practitioner.addName(convertToHumanName(name));
+    practitioner.addAddress(convertToAddress(addr, country));
+    practitioner.addTelecom(convertToContactPoint(ContactPointUse.WORK, telephone));
```

Comment:
Can you add chaining to all of the code changes to make it look nicer? The only thing is that `.setId()` returns a Resource, which would be annoying to chain so don't worry about that one. 
```java
    new Practitioner()
       .addName(convertToHumanName(name))
       .addAddress(convertToAddress(addr, country)
       .addTelecom(convertToContactPoint(ContactPointUse.WORK, telephone));
```